docs/guides/ssr-and-hydration.md

```diff
@@ -0,0 +1,198 @@
+---
+title: SSR and Hydration
+nav: 20
+---
+
+## Server Side Rendering (SSR)
+
+Sever Side Rendering (SSR) is a technique that helps us render our components into
+HTML strings on the server, send them directly to the browser, and finally "hydrate" the
+static markup into a fully interactive app on the client.
+
+### React
+
+Let's say we want to render a stateless app using React. In order to do that, we need
+to use `express`, `react` and `react-dom/server`. We don't need `react-dom/client`
+since it's a stateless app.
+
+Let's dive into that:
+
+- `express` helps us build a web app that we can run using Node,
+- `react` helps us build the UI components that we use in our app,
+- `react-dom/server` helps us render our components on a server.
+
+```json
+// package.json
```

Comment:
It looks like a `tsconfig.json`, not `package.json`. TSConfig [is not currently supported in package.json](https://github.com/microsoft/TypeScript/issues/32830).

```suggestion
// tsconfig.json
```